### DISCUSSION TITLE
Can we flip the page without changing header/footer positioning?

### DISCUSSION THREAD

**Discussion by phongnt570** [typst/typst#4138] [Q&A]
Currently the `flipped` parameter would switch the page from portrait to landscape mode, then the header and footer would be moved to the long edge of the paper accordingly. How could I flip the page and keep the original positioning of header/footer? Something that looks like this (page 64):

<img width="706" alt="image" src="https://github.com/typst/typst/assets/6749421/8da4d508-1918-4425-af13-e737012bb154">

I was able to rotate the main content with the `rotate` function, but the page still stays portrait in the output PDF which is not nice for reading.

Any helps would be much appreciated!

**Comment by l0wx** [comment]
No, but you could hackily replicate that desired behavior by interchanging the `.margin`s as well as the  `rotate()`ing & `place()`ing `.header` & `.footer` by hand.

**Reply by burrbull** [reply]
Could you show how to do this?
